.github/workflows/cron.yaml

```diff
@@ -45,4 +45,13 @@ jobs:
             git add .
             git commit -m "Update VEX documents"
             git push
-          fi
\ No newline at end of file
+          fi
+
+  keepalive-job:  # needed to keep GitHub from disabling the actions due to inactivity
+    name: Keepalive Workflow
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gautamkrishnar/keepalive-workflow@v2
```

Comment:
The repo that contains this action seems to be disabled by GH due to a possible TOS violation 🤷🏻 

In order to simplify the workflow and reduce dependencies (and possible supply chain risks), perhaps wouldn't be simpler to run inside [aquasecurity/vexhub](https://github.com/aquasecurity/vexhub)?

Reply:
Opened https://github.com/aquasecurity/vexhub/pull/6

Reply:
We merged https://github.com/aquasecurity/vexhub/pull/6 and this works - https://github.com/aquasecurity/vexhub/actions/runs/15552217353/job/43784857685
So, UUIC, we can remove cron workflow from vexhub-crawler